src/main/java/at/ac/tuwien/kr/alpha/common/IntIterator.java

```diff
@@ -0,0 +1,20 @@
+package at.ac.tuwien.kr.alpha.common;
+
+/**
+ * An iterator returning raw int integers instead of Integer objects (for efficiency).
+ *
+ * Copyright (c) 2020, the Alpha Team.
+ */
+public interface IntIterator {
+
+	/**
+	 * @return true if the iterator has more elements.
+	 */
+	public abstract boolean hasNext();
```

Comment:
Public interface methods should have no visibility modifier and no `abstract` since they are `public abstract`by default. See: https://stackoverflow.com/questions/5418464/methods-visibility-in-interface

Reply:
Fixed.